libs/agno/agno/app/fastapi/app.py

```diff
@@ -86,7 +86,13 @@ def get_router(self) -> APIRouter:
     def get_async_router(self) -> APIRouter:
         return get_async_router(agent=self.agent, team=self.team)
 
-    def get_app(self, use_async: bool = True, prefix: str = "/v1") -> FastAPI:
+    def get_agui_router(self) -> APIRouter:
```

Comment:
No we should override the sync and async router retrieve functions, like we don with whatsapp, etc.